src/MoBi.R/Services/ProjectTask.cs

```diff
@@ -0,0 +1,56 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using MoBi.Core.Domain.Model;
+using MoBi.Core.Serialization.ORM;
+using OSPSuite.Core.Domain;
+using OSPSuite.R.Domain;
+
+namespace MoBi.R.Services
+{
+   public interface IProjectTask
+   {
+      MoBiProject GetProject(string fileName);
```

Comment:
Is this on purpose that we had "GET" everywhere? We started to not do this anymore a long time ago, at least not systematically. Wondering about consistency here